organize/models.py

```diff
@@ -49,17 +49,29 @@ def create(self, **data_dict):
         )
 
         if previous_event:
-            event_date = data_dict["date"]
-            if date(event_date.year, event_date.month, event_date.day) - date(
-                previous_event.date.year, previous_event.date.month, previous_event.date.day
-            ) < timedelta(days=180):
-                raise ValidationError(
-                    {
-                        "date": _(
-                            "Your workshops should be at least 6 months apart. " "Please read our Organizer Manual."
-                        )
-                    }
-                )
+            try:
+                event_date = data_dict["date"]
```

Comment:
I'd move this outside of the `try` because it's used by the `try` and `except`.

Reply:
Noted and done.

Reply:
@amakarudze I can't see any changes pushed. Will keep an eye out tomorrow 👍

Reply:
Sorry I forgot to push the commit. Just did @marksweb.